cmdget.go

```diff
@@ -14,8 +15,8 @@ import (
 )
 
 var getCommand = &Command{
-	UsageLine: "get [-vcs] [-u] [-f] [module...]",
-	Short:     "start hacking a module",
+	UsageLine: "get [-vcs] [-u] [-f] [module/package...]",
+	Short:     "start hacking a module or package",
```

Comment:
Technically, we can't hack a single package, so perhaps the original description is still correct? We could say something like:

```
The get command checks out Go module dependencies into a directory
where they can be edited. If a package is named, its containing module
is copied.
```
?

Reply:
Done, with minor modifications.